core/templates/dev/head/pages/exploration_editor/editor_tab/TrainingDataServiceSpec.js

```diff
@@ -115,63 +113,24 @@ describe('TrainingDataService', function() {
     });
 
     ecs.setActiveStateName('State');
-
-    $httpBackend.when('GET', '/createhandler/training_data/0/State').respond({
-      unhandled_answers: [{
-        answer: 'answer1',
-        frequency: 2
-      }, {
-        answer: 'answer2',
-        frequency: 1
-      }]
-    });
   }));
 
-  afterEach(function() {
-    $httpBackend.verifyNoOutstandingExpectation();
-    $httpBackend.verifyNoOutstandingRequest();
-  });
-
-  it('should call a backend handler to initialize training data', function() {
```

Comment:
Why delete?

Reply:
We don't have backend controllers for training data. It is integral part of exploration itself.

Reply:
Ah yes, OK.